test_utils.go

```diff
@@ -0,0 +1,222 @@
+package viamrtsp
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/bluenviron/gortsplib/v4"
+	"github.com/bluenviron/gortsplib/v4/pkg/base"
+	"github.com/bluenviron/gortsplib/v4/pkg/description"
+	"github.com/bluenviron/gortsplib/v4/pkg/format"
+	"github.com/bluenviron/mediacommon/pkg/codecs/h264"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/test"
+	"go.viam.com/utils"
+)
+
+// NewH264ServerHandler creates a new H264 server handler for testing.
+func NewH264ServerHandler(
```

Comment:
Created `test_utils.go` so that we could export fake RTSP helpers for use in the viamonvif package. (Cannot export across packages with `_test` files)